src/System.Net.Requests/tests/HttpWebRequestTest.netstandard1.7.cs

```diff
@@ -0,0 +1,28 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
The document is giving examples only for non-version pivots. My question is about version pivots such as netstandard1.3, 1.7/2.0 and future v3, v4, v5, etc. 
/cc @karelz @ericstj @stephentoub 

What is the benefit for testing netstandard1.3 and netstandard1.7 at this point? Isn't 1.3 a subset of 1.7? 
Maintaining all backward versions will soon create a very complex set of tests that will be hard to maintain and understand. E.g.:

> Uh, thank you. I still find this confusing.